src/region/__tests__/RegionAnnotations-test.tsx

```diff
@@ -79,23 +79,27 @@ describe('RegionAnnotations', () => {
         });
 
         describe('handleStop', () => {
-            const shape = {
+            const definedShape = {
```

Comment:
Wasn't too sure what to name this, I couldn't just call it `shape` because we have `shape` already being used in the test.each block.